streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -106,13 +108,20 @@
     KStream<K, V> toStream();
 
     /**
+     * Creates a new instance of KStream from this KTable
+     *
+     * @return the instance of KStream
+     */
+    <K1> KStream<K1, V> toStream(ValueMapper<K, K1> mapper);
```

Comment:
This function maps the key instead of the values, so it is actually the same to ```table.toStream().map(mapper)``` where ```mapper``` should not change value. I feel this is a nice syntax-sugar to have.